src/index.js

```diff
@@ -20,10 +20,10 @@ if (typeof process === 'undefined' || !process.env?.RTL_SKIP_AUTO_CLEANUP) {
     teardown(() => {
       cleanup()
     })
-  } else if (!process.env.RTL_AFTEREACH_WARNING_LOGGED) {
+  } else if (process?.env && !process.env.RTL_AFTEREACH_WARNING_LOGGED) {
```

Comment:
If `process.env` is not defined, we'll not provide this functionality at all since we won't have a place to save the flag.